### PR TITLE
Upgraded Cucumber version and added screenshots to spark report

### PR DESCRIPTION
**Pull request by shomiruddin94**

### CODE REVIEW

File: src/test/java/AnalysisTools/AnalysisToolsRunners/RunComets2ProgressionTest.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/AnalysisToolsRunners/RunComets2ProgressionTest.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/AnalysisToolsRunners/RunComets2SmokeTest.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/AnalysisToolsRunners/RunComets2SmokeTest.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/AnalysisToolsRunners/RunICRPSmokeTest.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/AnalysisToolsRunners/RunICRPSmokeTest.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/AnalysisToolsRunners/RunLDLinkRegressionTest.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/AnalysisToolsRunners/RunLDLinkRegressionTest.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/AnalysisToolsRunners/RunLDLinkSmokeTest.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/AnalysisToolsRunners/RunLDLinkSmokeTest.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/AnalysisToolsRunners/RunMSigPortalProgressionTest.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/AnalysisToolsRunners/RunMSigPortalProgressionTest.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/AnalysisToolsRunners/RunMSigPortalSmokeTest.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/AnalysisToolsRunners/RunMSigPortalSmokeTest.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/AnalysisToolsRunners/RunPLCOProgressionTest.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/AnalysisToolsRunners/RunPLCOProgressionTest.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/AnalysisToolsRunners/RunPLCOSmokeTest.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/AnalysisToolsRunners/RunPLCOSmokeTest.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/CEDCD/Steps/CEDCDAdminPageSteps.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/CEDCD/Steps/CEDCDAdminPageSteps.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/CEDCD/Steps/CEDCDBiospecimenCountClearAllSteps.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/CEDCD/Steps/CEDCDBiospecimenCountClearAllSteps.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/CEDCD/Steps/CEDCDBiospecimenCountsAlphabeticalCancerTypeSteps.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/CEDCD/Steps/CEDCDBiospecimenCountsAlphabeticalCancerTypeSteps.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/CEDCD/Steps/CEDCDBiospecimenCountsSelectAllCohortsSteps.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/CEDCD/Steps/CEDCDBiospecimenCountsSelectAllCohortsSteps.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/CEDCD/Steps/HooksSteps.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/CEDCD/Steps/HooksSteps.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/Comets2/Steps/HooksSteps.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/Comets2/Steps/HooksSteps.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/ICRP/Steps/HooksSteps.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/ICRP/Steps/HooksSteps.java:1]: Please see commented-out imports

File: src/test/java/AnalysisTools/ICRP/Steps/ICRPSearchSteps.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/ICRP/Steps/ICRPSearchSteps.java:1]: Please see commented-out line 37-38

File: src/test/java/AnalysisTools/JPSurv/Steps/JPSurvHomePageSteps.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/JPSurv/Steps/JPSurvHomePageSteps.java:1]: Please see commented-out line 34-41, 111-112, 140-141

File: src/test/java/AnalysisTools/cProSite/Pages/CProSiteExplorePage.java
Comment (georgeb79) [inline src/test/java/AnalysisTools/cProSite/Pages/CProSiteExplorePage.java:1]: Locators missing description

File: src/test/java/CustomBusiness/BusinessAppRunners/RunAppEIDPProgressionTest.java
Comment (georgeb79) [inline src/test/java/CustomBusiness/BusinessAppRunners/RunAppEIDPProgressionTest.java:1]: Please see commented-out imports

File: src/test/java/CustomBusiness/DCEG/Steps/RegularUserFlowSteps.java
Comment (georgeb79) [inline src/test/java/CustomBusiness/DCEG/Steps/RegularUserFlowSteps.java:1]: Please see commented-out imports

File: src/test/java/CustomBusiness/DCEG/StepsImplementation/FlowStepsImplementation.java
Comment (georgeb79) [inline src/test/java/CustomBusiness/DCEG/StepsImplementation/FlowStepsImplementation.java:1]: Please see commented-out imports

File: src/test/java/CustomBusiness/EIDP/Pages/IDPAwaitingResponsePage.java
Comment (georgeb79) [inline src/test/java/CustomBusiness/EIDP/Pages/IDPAwaitingResponsePage.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ATO/Pages/BasePage.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ATO/Pages/BasePage.java:1]: Please see commented-out lines 199, 204 to 212, 217, 220, 222

File: src/test/java/ServiceNow/ATO/Pages/SecurityFormPage.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ATO/Pages/SecurityFormPage.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ATO/Steps/HookSteps.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ATO/Steps/HookSteps.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ATO/Utils/ReportUtil.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ATO/Utils/ReportUtil.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/CADSR/Pages/BasePage.java
Comment (georgeb79) [inline src/test/java/ServiceNow/CADSR/Pages/BasePage.java:1]: Please see commented-out lines 231, 236-244, 249, 252

File: src/test/java/ServiceNow/CHARMS/Steps/MelanomaScreenerSubmission.java
Comment (georgeb79) [inline src/test/java/ServiceNow/CHARMS/Steps/MelanomaScreenerSubmission.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/CHARMS/Steps/RASSurveySteps.java
Comment (georgeb79) [inline src/test/java/ServiceNow/CHARMS/Steps/RASSurveySteps.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/CICDBuildRunners/Run_CICD_DevOps_Application_Build_Test.java
Comment (georgeb79) [inline src/test/java/ServiceNow/CICDBuildRunners/Run_CICD_DevOps_Application_Build_Test.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/CICDBuildRunners/Run_CICD_DevOps_UpdateSet_Build_Test.java
Comment (georgeb79) [inline src/test/java/ServiceNow/CICDBuildRunners/Run_CICD_DevOps_UpdateSet_Build_Test.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/CICDBuildRunners/Run_CICD_Integration_Application_Build_Test.java
Comment (georgeb79) [inline src/test/java/ServiceNow/CICDBuildRunners/Run_CICD_Integration_Application_Build_Test.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/CICDBuildRunners/Run_ESR_Application_Build_Test.java
Comment (georgeb79) [inline src/test/java/ServiceNow/CICDBuildRunners/Run_ESR_Application_Build_Test.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/CICDBuildRunners/Run_MyCICDTestApp_Application_Build_Test.java
Comment (georgeb79) [inline src/test/java/ServiceNow/CICDBuildRunners/Run_MyCICDTestApp_Application_Build_Test.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/CHARMS/Steps/myRASScreenerSubmissions.java
Comment (georgeb79) [inline src/test/java/ServiceNow/CHARMS/Steps/myRASScreenerSubmissions.java:1]: Please see commented-out lines 104, 1570-1574, 1870-1872, 2350-2356, and 2642-2652

File: src/test/java/ServiceNow/CcrHelpDesk/Steps/HookSteps.java
Comment (georgeb79) [inline src/test/java/ServiceNow/CcrHelpDesk/Steps/HookSteps.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ITPG/Pages/BasePage.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ITPG/Pages/BasePage.java:1]: Please see commented-out  lines 217-225, 230, and 233

File: src/test/java/ServiceNow/ITPG/Steps/HookSteps.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ITPG/Steps/HookSteps.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/NERD/Steps/HooksSteps.java
Comment (georgeb79) [inline src/test/java/ServiceNow/NERD/Steps/HooksSteps.java:1]: Please see commented-out imports

File: src/main/java/com/nci/automation/web/CommonUtils.java
Comment (matakevin) [inline src/main/java/com/nci/automation/web/CommonUtils.java:1]: Is there a reason for the "//" slashes on lines 82-149?

File: src/test/java/ServiceNow/RCMDB/Steps/HookSteps.java
Comment (georgeb79) [inline src/test/java/ServiceNow/RCMDB/Steps/HookSteps.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/SampleTest/Steps/HooksSteps.java
Comment (georgeb79) [inline src/test/java/ServiceNow/SampleTest/Steps/HooksSteps.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunAppTrackerProgressionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunAppTrackerProgressionTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunAtoProgressionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunAtoProgressionTest.java:1]: Please see commented-out imports and line 21

File: src/test/java/ServiceNow/ServiceNowRunners/RunAtoRegressionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunAtoRegressionTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunCADSRServicePortalTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunCADSRServicePortalTest.java:1]: Please see commented-out imports and line 21

File: src/test/java/ServiceNow/ServiceNowRunners/RunCCRHelpDesk.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunCCRHelpDesk.java:1]: Please see commented-out imports and line 21

File: src/test/java/ServiceNow/ServiceNowRunners/RunCHARMSProgressionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunCHARMSProgressionTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunCHARMSRegressionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunCHARMSRegressionTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunCHARMSSmokeTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunCHARMSSmokeTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunCHARMSTestAccountResetTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunCHARMSTestAccountResetTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunCOVIDCodeProgressionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunCOVIDCodeProgressionTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunCOVIDDashProgressionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunCOVIDDashProgressionTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunCOVIDDashRegressionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunCOVIDDashRegressionTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunCOVIDDashSmokeTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunCOVIDDashSmokeTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunITPGPSmokeTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunITPGPSmokeTest.java:1]: Please see commented-out imports and line 21

File: src/test/java/ServiceNow/ServiceNowRunners/RunITPGProgerssionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunITPGProgerssionTest.java:1]: Please see commented-out imports and line 21

File: src/test/java/ServiceNow/ServiceNowRunners/RunITPGProgressionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunITPGProgressionTest.java:1]: Please see commented-out imports and line 21

File: src/test/java/ServiceNow/ServiceNowRunners/RunNERDProgressionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunNERDProgressionTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunNERDRegressionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunNERDRegressionTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunNERDSmokeTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunNERDSmokeTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunRCMDBTestCases.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunRCMDBTestCases.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunSEERProgressionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunSEERProgressionTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunSEERRegressionTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunSEERRegressionTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunSEERSmokeTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunSEERSmokeTest.java:1]: Please see commented-out imports

File: src/test/java/ServiceNow/ServiceNowRunners/RunSampleRunnerTest.java
Comment (georgeb79) [inline src/test/java/ServiceNow/ServiceNowRunners/RunSampleRunnerTest.java:1]: Please see commented-out imports

**Review by georgeb79 — CHANGES_REQUESTED** [review]

File: src/main/java/com/nci/automation/web/CommonUtils.java
Comment (matakevin) [reply src/main/java/com/nci/automation/web/CommonUtils.java:1]: intelliJ adds the "//" to format correctly.  Disregard aforementioned comment

**Review by georgeb79 — APPROVED** [review]

**Review by sarwarahmed1 — APPROVED** [review]

**Review by iamdez99 — APPROVED** [review]
Reviewed and approved

**Pull request merged**